components/project-card/project-card.jsx

```diff
@@ -149,7 +149,7 @@ export default React.createClass({
                   : (<div>
                       <Link to={detailViewLink} onClick={this.handleReadMoreClick}>Read more</Link>
                     </div>);
-    let creators = this.props.creators ? `By ${this.props.creators.join(`, `)}` : null;
+    let creators = this.props.creators.length > 0 ? `By ${this.props.creators.join(`, `)}` : null;
```

Comment:
yea I can add `PropTypes` to `Details` but we might still need to rely on `this.props.creators.length > 0` to decide if we should render the "By _____" line or not. (as `creator` can be an empty array)